cfme/common/__init__.py

```diff
@@ -499,12 +502,12 @@ def get_tags(self, tenant="My Company Tags"):
             # check for users/groups page in case one tag string is returned
             for tag in [tags_text] if isinstance(tags_text, six.string_types) else list(tags_text):
                 tag_category, tag_name = tag.split(':')
-                category = self.appliance.collections.categories.instantiate(
-                    display_name=tag_category)
-                tags.append(category.collections.tags.instantiate(
-                    display_name=tag_name.strip(),
-                    name=tag_category.strip()))
-        return tags
+                # instantiate category first, then use its tags collection to instantiate tag
+                tags_objs.append(
+                    self.appliance.collections.categories.instantiate(display_name=tag_category)
```

Comment:
Category is description right?

Reply:
Can you elaborate? The `TagCollection` has a `Category` as its parent, and the `Tag` has `name` and `display_name` attributes.

The attributes aren't named very well, but I'm not trying to change them in this scope.  MIQ uses `name` and `description`, where `name` is more akin to a key, and `description` is the displayed tag value.

Reply:
Never mind, this was in reference to our conversation offline about categories/name/description.

Reply:
Yeah there's a bit of a mismatch between the Tag/Category class attributes and what those fields are called in CFME. That should be pretty straightforward to change.

What's more complex is fixing the `get_tags()` method to resolve the class attributes for those tags that aren't displayed (name) when viewing tags on an entity.  Not for this PR :)